python/docs/source/doc_maintenance.rst

```diff
@@ -0,0 +1,66 @@
+Document Maintenance
+====================
+
+Document Software Setup
+-----------------------
+
+Vowpal Wabbit python Wrapper documentation is built using Sphinx.
+See http://www.sphinx-doc.org/en/master/index.html
+
+
+You can create, build, and preview the documentation on your local machine.
+
+You can commit your updates to the vowpal wabbit repository on GitHub, which triggers an automatic rebuild.
+
+
+Clone Vowpal Wabbit
+-------------------
+
+To get started, create or identify a working directory on your local machine.
```

Comment:
Documentation is built off of the currently installed version of the vowpalwabbit package. Doing these steps will not work right?

Reply:
For the current master branch, it is working.

Reply:
I just verified then that this is not the case. The package must be installed to build the documentation. Without it the command `make html` will technically exit with a status code of 0, but the built documentation will be empty.

Reply:
My bad, I have installed the required package so for me it is working, I will add the requirements.txt and required command to install the packages.

Reply:
Describing how to build the documentation would be better suited in the wiki. Because of the fact there are bindings in several languages I think we should opt to keeping auto generated docs to just describing the code itself and examples for now. I think this page makes sense: https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Python

Reply:
okay, I will remove from here and add a wiki page for the same.